tests/Api/Request/ParametersBuilderTest.php

```diff
@@ -83,7 +83,35 @@ public function setSearchParamsProvider()
                         0 => 'variation_id'
                     ],
                     'attrib' => [
-                        'color' => ['red', 'blue']
+                        'color' => ['red', 'blue'],
+                        'cat' => ['Category 2']
+                    ],
+                    'selected' => ['cat' => ['Category']],
+                    'order' => 'price ASC',
+                    'count' => 0,
+                    'first' => 0
+                ]
+            ],
+            'Category page request with subcategory selected' => [
+                'parameters' => [
+                    Plugin::API_PARAMETER_ATTRIBUTES => [
```

Comment:
Isn't `attrib` as a string used everywhere else? I typically prefer to not use constants in unit-tests, as if the constant changes in the future, you can see which components would be affected.